deployment/conf/supported_apps_w_extensions.json

```diff
@@ -117,6 +117,12 @@
       "xls",
       "xlsx"
     ],
+    "upload_specification": [
+      "csv",
+      "tsv",
+      "xls",
+      "xlsx"
```

Comment:
open office also uses `ods`  (OpenDocument Spreadsheet Document format) which might be easily supported.

Reply:
There's also `xlsm` apparently. If we want to support more formats that's easy enough, but let's do it in another PR.

Reply:
I don't think xlsm needs to be included. I don't think we came across that during user testing as a common file type. Only if it is part of a database like SESAR or ESS-DIVE or NMDC, then maybe to include down the line.

Reply:
I don't recall any .xlsm, and I honestly can't think of a use case for them when creating an import specification file. I wouldn't recommend much effort into file selection because it's pretty trivial for a user to open a file in Excel and Save As CSV, TSV, or XLS(X), even before accounting for the fact that the overwhelming majority of files will already be in those formats.